egs/librispeech/asr/simple_v1/train.py

```diff
@@ -26,6 +26,7 @@
 from snowfall.models import AcousticModel
 from snowfall.models.tdnn_lstm import TdnnLstm1b
 from snowfall.training.graph import TrainingGraphCompiler
+from snowfall.training.ctc_graph import CtcTrainingGraphCompiler
```

Comment:
I think you forgot to check this file in

Reply:
Thanks, added.